_posts/2017-01-30-inc-cmd-lessonslearned.md

```diff
@@ -0,0 +1,79 @@
+category:   Incident Response
+tags:       Incident management, Incident command
+author:     patrick_holmes
+image:      peace_war_inccmd.jpg
+---
+
+Incident command is a reasonably new area of focus for SBG. In a nutshell we have a nominated technical person known as the Incident Commander (IC) who gives direction in order to resolve an incident and restore service as quickly as possible.
+
+This blog post contains some of the insights and 'lessons learned' by our teams from their experiences in live incidents and exercises (known internally as fire drills) as they work to improve their skills and reduce our MTTR. The methodology for this is largely forked from O'Reilly's highly recommended [Incident Management for Operations](http://shop.oreilly.com/product/0636920036159.do) book, influences from SAR/British Army doctrine, with some additions based on some of our experiences across the company.
+
+***
+
+#### DON'T PANIC!
+It’s just a computer system and a technical problem that needs solving, the only difference between this and a JIRA story is the manner of approach and doing it with urgency. Take a ‘Condor moment’ then crack on with systematically triaging your way through the issues and bringing the impacted services back online.
+
+#### The first responder is the first incident commander
+The first person on the scene is by default in control of the incident, until until they get more help or can safely hand it off to someone else. In some cases this may mean they are best focusing on raising the alarm rather than immediately fighting the fire, but in all cases they need to focus effort to gain control of the situation.
+
+#### Seize the initiative
+Make the mental switch from ‘peace’ to ‘war’ and do not allow the situation and events that are occurring to dictate the pace and direction of the incident. A calm and confident incident commander takes control of the situation and issues clear and explicit directives to his or her available technical resource in order to triage the incident and drive it towards resolution as quickly and safely as possible with the minimum amount of collateral damage. This is often the difference between a ten minute incident and a two hour incident.
+
+#### IC’s do not fix problems
+As coordinators of resource the IC should not be at the terminal issuing commands. It is the role of the IC to set the strategic tone for the incident and leave the individual tactics to the subject matter experts at their disposal. In many cases, the IC has the team debate the pro's and con's of a particular solution – but the IC makes the final call based on the best available information to them from the SME's at the time.
```

Comment:
"pros and cons" - no apostrophe necessary.

SME is undefined - again consider an `abbr` or expanding it followed by the abbreviation in parens.